sql/pgwire/v3.go

```diff
@@ -172,6 +172,7 @@ func (c *v3Conn) serve(authenticationHook func(string, bool) error) error {
 	for key, value := range map[string]string{
 		"client_encoding": "UTF8",
 		"DateStyle":       "ISO",
+		"server_version":  "9.1.0",
```

Comment:
Yeah, I agree - I wanted to get some discussion about which version we should report. I'm leaning towards 9.5.0 for now, since that's the latest at the time of writing.